src/main/java/edu/tamu/sage/model/response/Entry.java

```diff
@@ -0,0 +1,42 @@
+package edu.tamu.sage.model.response;
+
+public class Entry {
+
```

Comment:
No default constructor?
I think this should have a default constructor beyond the static constructor defined below (`of()`).